### PR TITLE
VideoLAN.VLC version 3.0.18

### PR DESCRIPTION
**Pull request by borque**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90601)

### CODE REVIEW

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+- vob
+- webm
+- wma
+- wmv
+- zab
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.msi
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:56]:
```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.msi
```
Please use the direct URL to the installer.

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  ProductCode: '{B022B1C5-D067-42CB-98E2-D965E4D74CFE}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player 3.0.18 (64-bit)
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
+    InstallerType: msi
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.msi
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:67]:
```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.msi
```
Please use the direct URL to the installer.

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  InstallerSha256: 72EB05D21A4159450DBB39F2A09C5ED7C8370C9B4CAD75CE36A1A71B9F8A7D11
+  ProductCode: '{6799A970-32B3-47B4-89AA-2DF0806FB862}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player 3.0.18 (32-bit)
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
+    InstallerType: msi
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.exe
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:77]:
```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.exe
```
Please use the direct URL to the installer.

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+  InstallerSha256: BA575F153D357EAF3FDBF446B9B93A12CED87C35887CDD83AD4281733EB86602
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
+    InstallerType: nullsoft
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.exe
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:86]:
```suggestion
  InstallerUrl: https://download.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.exe
```
Please use the direct URL to the installer.

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- zab
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.msi
+  InstallerSha256: A618D66B2F753343402200AD17D92EB6D4AB10DB0DD1A098402E2968AA3114C4
+  ProductCode: '{B022B1C5-D067-42CB-98E2-D965E4D74CFE}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player 3.0.18 (64-bit)
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:60]:
```suggestion
  - DisplayName: VLC media player
```
Must match Apps&Features/Control Panel.

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+    DisplayVersion: 3.0.18
+    InstallerType: msi
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: wix
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.msi
+  InstallerSha256: 72EB05D21A4159450DBB39F2A09C5ED7C8370C9B4CAD75CE36A1A71B9F8A7D11
+  ProductCode: '{6799A970-32B3-47B4-89AA-2DF0806FB862}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player 3.0.18 (32-bit)
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:71]:
```suggestion
  - DisplayName: VLC media player
```
Must match Apps&Features/Control Panel.

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:73]:
```suggestion
    DisplayVersion: 3.0.18.0
```
Must match Apps&Features/Control Panel.

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+    InstallerType: msi
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.exe
+  InstallerSha256: BA575F153D357EAF3FDBF446B9B93A12CED87C35887CDD83AD4281733EB86602
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:82]:
```suggestion
```
DisplayVersion should not be used when it is equal to PackageVersion.

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+    InstallerType: nullsoft
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win32/vlc-3.0.18-win32.exe
+  InstallerSha256: F4BAAA8135E0F9A993F0258A4D095DB475096896BD3ADB48369F1F70C1F0D9D4
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player
+    Publisher: VideoLAN
+    DisplayVersion: 3.0.18
```
Comment (mdanish-kh) [inline manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:91]:
```suggestion
```
DisplayVersion should not be used when it is equal to PackageVersion.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml (hunk 5 above)
Comment (borque) [reply manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml:60]: Ahh, allright, I "reused" this from the previous manifest and simply adjusted the version number...

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**